### PR TITLE
add task product-cart

### PR DESCRIPTION
**Pull request by InnaSh23**
[DEMO LINK](https://InnaSh23.github.io/layout_product-cards/)
[TEST REPORT LINK](https://InnaSh23.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,97 @@
...
+
+        <p class="card__price-value">$2,199</p>
+      </div>
+
+      <a
+        class="card__button card__button--active"
+        data-qa="hover"
+        href="#"
+        >Buy
+      </a>
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html:94]:
```suggestion
        href="#"
      >
        Buy
      </a>
```

File: src/styles/blocks/card.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
+/* #region card */
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/card.css:1]: remove comments

File: src/styles/blocks/card.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  font-family: Roboto, sans-serif;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/card.css:9]: set font family for html tag and inherit it if needed

File: src/styles/blocks/card.css [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  font-weight: 400;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/card.css:10]: remove default preperty

File: src/styles/blocks/stars.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
+/* #region Stars */
+:root {
+  --star-active: url("../../images/star-active.svg") center no-repeat;
+}
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/stars.css:4]: move into variables.css file

File: src/styles/index.css [hunk 6]
```diff
@@ -0,0 +1,10 @@
+@import "./blocks/card.css";
+@import "./blocks/stars.css";
+
+/* #region reset */
+body,
+h2,
+p {
+  margin: 0;
+}
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/index.css:9]: move into reset.css

File: src/styles/blocks/card.css [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-name {
+  font-size: 12px;
+  color: #616070;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/card.css:56]: consider moving all common colors into a variables

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,98 @@
...
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="stars stars--1">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
```
Comment (vlad-demchuk) [inline src/index.html:43]: why do we need to repeat all stars blocks?

it should work with only one

File: src/styles/blocks/card.css [hunk 9]
```diff
@@ -0,0 +1,86 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: var(--color-main-text);
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.css:10]: move `box-sizing` to `*` selector

File: src/styles/blocks/card.css [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+  background-color: #fff;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  margin-bottom: 40px;
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.css:22]: use shorthand css property

File: src/styles/blocks/card.css [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+  background-color: var(--color-for-button);
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+}
+
+.card__button--active:hover {
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.css:81]: redundant class

`card__button:hover` instead

File: src/styles/blocks/stars.css [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+.stars--5 :nth-child(n) {
+  background: var(--star-active);
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--5 {
+  display: none;
```
Comment (vlad-demchuk) [inline src/styles/blocks/stars.css:28]: no need to use this, leave only one block with stars

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/styles/blocks/card.css (hunk 10 above)
Comment (InnaSh23) [reply src/styles/blocks/card.css:22]: yes, I overlooked it

File: src/styles/blocks/stars.css [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+  background: url("../../images/star.svg") center no-repeat;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--4 :not(:last-child) {
+  background: var(--star-active);
+}
```
Comment (Viktor-Kost) [inline src/styles/blocks/stars.css:17]: You set stars--4 at html, however you need to be flexible and if you get a card with 3 stars, you will just set stars--3 and you've already have styles for this case.
So, just move star styles from previous task here

File: src/styles/blocks/card.css [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+.card__price-value {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__button {
+  width: 166px;
+  height: 40px;
```
Comment (Viktor-Kost) [inline src/styles/blocks/card.css:70]: it's better not hardcode height for elements with content inside. Just use paddings

File: src/styles/blocks/stars.css (hunk 13 above)
Comment (InnaSh23) [reply src/styles/blocks/stars.css:17]: I did this in a commit [a0014d6](https://github.com/mate-academy/layout_product-cards/pull/2914/commits/a0014d6e71117103d6f32e818fd849e88171ca90), but the mentor told me to do one block

**Pull request closed without merging**